### ISSUE TITLE
feat: add [WordPress] to/in [integration]

### ISSUE THREAD

**Issue by A55imilat0r** [microsoft/fast#5526]
# 🙋 Feature Request

I'm looking for someone to create a Theme and Plugin that will bring Fluent UI to Wordpress

## 🤔 Expected Behavior

Full drag and drop site builder with Fluent UI assets. Buttons, Styles, Blocks, Colours ect.... 

## 😯 Current Behavior

There is no Theme or Plugin in wordpress that makes Fluent UI a simple low/no code option. 

## 💁 Possible Solution

I'm open to community suggestions

## 🔦 Context

No/low code Fluent UI Theming in WordPress

Not being able to create a website that looks nice on Windows 10/11 

Learning how to code JS but it's not my area of expertise. 

## 💻 Examples

Basically turn this react-UWP concept site into a WordPress Theme and editor with all the options inside WordPress Theme editor admin console as a No/Low Code drag and drop for visual theme editor for wordpress.   
https://www.react-uwp.com/ 
https://github.com/myxvisual/react-uwp

**Comment by EisenbergEffect** [comment]
I think this is a very reasonable request and something we'd love to have. Unfortunately, I don't think we have the ability to tackle this one ourselves right now. This is a great opportunity for community contribution.

**Comment by A55imilat0r** [comment]
Thank you for the validation, I'm here and on the Discord and always open to questions, ideas and able to provide a live Wordpress Host domain for community testing.

**Comment by A55imilat0r** [comment]
Not stale still seeking community support to run with this.

**Comment by EisenbergEffect** [comment]
Thanks for tagging this @A55imilat0r. I've removed the stale tag. This remains something we are interested in and would love to do. At the present time, we aren't able to do the work ourselves, as much as we would like to. We'd be more than happy to work with a community member on this if someone wants to give it a shot.

**Comment by janechu** [comment]
Closing this due to lack of contributor interest.
